docs/userguide/configuring.rst

```diff
@@ -17,8 +17,8 @@ supercomputer at TACC.
 This config uses the `parsl.executors.HighThroughputExecutor` to submit
 tasks from a login node (`parsl.channels.LocalChannel`). It requests an allocation of
 128 nodes, deploying 1 worker for each of the 56 cores per node, from the normal partition.
-The config uses the `address_by_hostname()` helper function to determine
-the login node's IP address.
+To limit network connections to just the internal network the config specifies the address
+used by the infiniband interface with `address_by_interface('ib0')`
```

Comment:
> Warning, treated as error:
/home/runner/work/parsl/parsl/docs/userguide/configuring.rst:15:'any' reference target not found: address_by_interface('ib0')

Use double back-quotes here, to quote text - single backquotes will hyperlink to a target, which I guess worked because it didn't mind the empty `()` before.

Reply:
Fixed.